center/router/router_alert_cur_event.go

```diff
@@ -39,12 +39,35 @@ func parseAggrRules(c *gin.Context) []*models.AggrRule {
 	return rules
 }
 
+func getUserGroupIds(ctx *gin.Context, rt *Router, myGroups bool) ([]int64, error) {
+	if !myGroups {
+		return nil, nil
+	}
+	me := ctx.MustGet("user").(*models.User)
+	return models.MyGroupIds(rt.Ctx, me.Id)
+}
+
 func (rt *Router) alertCurEventsCard(c *gin.Context) {
 	stime, etime := getTimeRange(c)
 	severity := ginx.QueryInt(c, "severity", -1)
 	query := ginx.QueryStr(c, "query", "")
+	myGroups := ginx.QueryBool(c, "my_groups", false) // 是否只看自己组，默认false
+	gids, err := getUserGroupIds(c, rt, myGroups)
+
+	aggrID := ginx.QueryInt64(c, "aggr_id", 0)
+
+	alertEvent, err := models.GetAlertAggrViewByAggrID(rt.Ctx, aggrID)
+	ginx.Dangerous(err)
+
 	dsIds := queryDatasourceIds(c)
-	rules := parseAggrRules(c)
+
+	var format []*models.AggrRule
```

Comment:
The format variable is not expressive enough; use the previous rules variable instead